metadata.json

```diff
@@ -16,41 +16,62 @@
     }
   ],
   "operatingsystem_support": [
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "7",
+        "8"
+      ]
+    },
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
         "7"
       ]
     },
     {
-      "operatingsystem": "RedHat",
+      "operatingsystem": "OracleLinux",
       "operatingsystemrelease": [
-        "8"
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Scientific",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "18.04",
+        "20.04"
       ]
     },
     {
       "operatingsystem": "SLES",
       "operatingsystemrelease": [
-        "12"
+        "12",
+        "15"
       ]
     },
     {
-      "operatingsystem": "Debian",
+      "operatingsystem": "AlmaLinux",
       "operatingsystemrelease": [
-        "10"
+        "8"
       ]
     },
     {
-      "operatingsystem": "Ubuntu",
+      "operatingsystem": "RockyLinux",
       "operatingsystemrelease": [
-        "18.04"
+        "8"
       ]
     }
   ],
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 6.21.0 < 8.0.0"
+      "version_requirement": ">= 7.18.0 < 8.0.0"
```

Comment:
Why 7.18 and no older version of 7.x?

Reply:
As predominantly PE modules,  our CI only tests against the LTS PE and STS PE, this is an indicator of what we test against

Reply:
Perhaps influxdb we could just to 7, since it got an extended uses case